pkg/controllers/federatedresourcequota/federated_resource_quota_sync_controller.go

```diff
@@ -139,10 +135,7 @@ func (c *SyncController) cleanUpWorks(namespace, name string) error {
 		}
 	}
 
-	if len(errs) > 0 {
-		return errors.NewAggregate(errs)
-	}
-	return nil
```

Comment:
There are other files coded like this, updated together?

Reply:
Done.